site/docs/v1/tech/apis/identity-providers/links.adoc

```diff
@@ -0,0 +1,182 @@
+---
+layout: doc
+title: Link APIs
+description: The APIs for creating, retrieving, updating and deleting Identity Provider Links
+---
+
+== Overview
+
+[NOTE.since]
+====
+This API has been available since 1.28.0
+====
+
+This page contains the APIs that are used to manage Links that establish a relationship between a FusionAuth User and an Identity Provider.
+
+* <<Link a User>>
+* <<Complete a pending Link>>
+* <<Retrieve a Link>>
+* <<Unlink a User>>
+
+== Link a User
+
+This API is used to create a Link.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Link a User
+[.endpoint]
+.URI
+--
+[method]#POST# [uri]#/api/identity-provider/link#
+--
+
+==== Request Body
+
+[.api]
+[field]#identityProviderId# [type]#[UUID]# [optional]#Optional#::
+The Id of the identity provider.
+
+[field]#identityProviderUserId# [type]#[String]# [optional]#Optional#::
+The Id or unique identifier for the User that is provided by the identity provider.
+
+[field]#userId# [type]#[UUID]# [required]#Required#::
+The FusionAuth Id of the User that is being linked to the identity provider.
+
+[source,json]
+.Example Request JSON
+----
+include::docs/src/json/identity-providers/links/create-link-request.json[]
+----
+
+=== Response
+
+:never_search_error:
+include::docs/v1/tech/apis/_standard-post-response-codes.adoc[]
+:never_search_error!:
+
+include::docs/v1/tech/apis/identity-providers/_links-post-response-body.adoc[]
+
+== Complete a pending Link
+
+This API is used confirm a pending Link. If an identity provider is configured with a linking strategy of `Create a pending link`, a pending link Id will be returned from the Login API that can be used to complete the link.
```

Comment:
This is a little confusing.  It might be good to separate the concepts here that this API is called to "complete a pending link" between a user and an IdP.  The `pendingIdPLinkId` used to complete the link was obtained previously from a login with an IdP configured with a linking strategy of "Create a pending link".  Or something along those lines.  You might almost get away with reversing the two sentences and starting with "When an Idp is configured...".